dist/stickyfill.js

```diff
@@ -371,8 +371,8 @@
         updateScrollPos();
         initAll();
 
```

Comment:
Just a heads up that you need to add feature detection for passive events support before applying it to the event listener. Older browsers that don't support it will understand third object argument as truthy `capture` option.
Here's the snippet you can use borrowed from Modernizr:
```javascript
var passiveOption = undefined;
try {
  var opts = Object.defineProperty({}, 'passive', {
    get: function() {
      passiveOption = { passive: true };
    }
  });
  window.addEventListener('test', null, opts);
} catch (e) {}
win.addEventListener('scroll', onScroll, passiveOption);
win.addEventListener('wheel', onWheel, passiveOption);
```